### DISCUSSION TITLE
Some rsync options

### DISCUSSION THREAD

**Discussion by bigntallmike** [outdoorbits/little-backup-box#226] [Ideas]
There's no reason to support partial files and attempt changed-file rsync updates, so you should add the '-W' option to the rsync command-line which always transfers files as whole files instead. I also typically use '-u' to skip files that are newer on the backup drive. If for some random reason there's a matching but newer file on the backup, it means I've probably edited it and don't want it replaced with the camera card version. 
Aside: I've also moved the 'sync' command to before the LED is turned off so the LED is truly informative of when the process is complete.

**Comment by outdoorbits** [comment]
Thank you, I'll implement this, but give me a few weeks...

**Reply by bigntallmike** [reply]
No rush, its working on my end I just thought I'd share for everyone else.

**Comment by outdoorbits** [comment]
The -W option is implemented in the development branch and I got transfer rates from 40 to 120 MB/s, thank you! The -u option maybe is not safe as the raspberry might not alway has got the correct time.
